examples/tests/invalid/container_types.test.w

```diff
@@ -29,4 +29,10 @@ let a: Array<str> = MutArray<str>[];
 
 let mm1: MutMap<num> = { "a" => 1, "b" => 2, "c" => 3 };
 let mm2 = MutMap<num> { "a" => 1, "b" => 2, "c" => 3 };
-let mm3 = mm2.copyMut();
\ No newline at end of file
+let mm3 = mm2.copyMut();
+
+
+let ss1: MutSet<str> = Set{"c"};
```

Comment:
We normally add a comment with the expected error

Reply:
I'll update it.